apps/cevent_frontend/src/utils/interfaces/EventInterfaces.ts

```diff
@@ -24,3 +24,36 @@ export interface EventDetailDto extends EventHomePageDto {
   organizerUserId: number;
   createdAt: Date;
 }
+
+interface Event {
```

Comment:
Please consider not replacing the above existing event interfaces with a more general one.